packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -1514,7 +1514,11 @@ describeCompat("stashed ops", "NoCompat", (getTestObjectProvider, apis) => {
 		},
 	);
 
-	it("offline blob upload", async function () {
+	it("blob upload before loading", async function () {
+		// TODO: AB#19035: Blob upload from an offline state does not currently work before establishing a connection on ODSP due to epoch mismatch.
```

Comment:
Mismatch is misleading. My testing suggest service does throw an error if we don't send epoch with a 400 error, which is the case when loading offline, since we haven't made any connection yet before making the createBlob call. 
<img width="983" alt="image" src="https://github.com/user-attachments/assets/18a4fb72-9268-4889-b6fc-ebbecf1437d8">

Reply:
ok, makes sense now.